test/e2e/kuttl/ScaleUpAndDownDC/01-assert.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cassandra-e2e-dc1-rack1
+status:
+  currentReplicas: 2
+  replicas: 2
+---
+apiVersion: db.orange.com/v1alpha1
+kind: CassandraCluster
+metadata:
+  name: cassandra-e2e
+spec:
+  nodesPerRacks: 1
+  topology:
```

Comment:
No need to test the topology if you update the one at the root level

Reply:
Removed